crates/ruff_python_parser/src/snapshots/ruff_python_parser__lexer__tests__fstring_with_multiline_format_spec.snap

```diff
@@ -0,0 +1,175 @@
+---
+source: crates/ruff_python_parser/src/lexer.rs
+expression: lex_source(source)
+---
+[
+    (
+        FStringStart,
+        0..4,
+    ),
+    (
+        FStringMiddle {
+            value: "__",
+            is_raw: false,
+        },
+        4..6,
+    ),
+    (
+        Lbrace,
+        6..7,
+    ),
+    (
+        NonLogicalNewline,
+        7..8,
+    ),
+    (
+        Name {
+            name: "x",
+        },
+        12..13,
+    ),
+    (
+        Colon,
+        13..14,
+    ),
+    (
+        FStringMiddle {
+            value: "d\n",
+            is_raw: false,
+        },
+        14..16,
+    ),
+    (
+        Rbrace,
+        16..17,
+    ),
+    (
+        FStringMiddle {
+            value: "__",
+            is_raw: false,
+        },
+        17..19,
+    ),
+    (
+        FStringEnd,
+        19..22,
+    ),
+    (
+        Newline,
+        22..23,
+    ),
+    (
+        FStringStart,
+        23..27,
+    ),
+    (
+        FStringMiddle {
+            value: "__",
+            is_raw: false,
+        },
+        27..29,
+    ),
+    (
+        Lbrace,
+        29..30,
+    ),
+    (
+        NonLogicalNewline,
+        30..31,
+    ),
+    (
+        Name {
+            name: "x",
+        },
+        35..36,
+    ),
+    (
+        Colon,
+        36..37,
+    ),
+    (
+        FStringMiddle {
+            value: "a\n        b\n          c\n",
+            is_raw: false,
+        },
+        37..61,
```

Comment:
Does the parser handle treating this as a format spec?

Reply:
Yes, I'll add a test case for it.